include/termdb.hpp

```diff
@@ -1382,6 +1403,223 @@ std::string TermDb::parser(const std::string &s, param p1, param p2, param p3,
     return incorrectString ? "" : result;
 }
 
+#elif defined(OS_WIN)
+
+class TermDb {
+private:
+    std::string name;
+    bool isValidState    = false;
+    bool isLessThanWin10 = false;
+    HANDLE consoleHandle = nullptr;
+
+    void _emulate_clear() const noexcept;
+    void _emulate_cup(const short) const noexcept;
+    void _emulate_cud(const short) const noexcept;
+    void _emulate_cuf(const short) const noexcept;
+    void _emulate_cub(const short) const noexcept;
+    void _emulate_cha(const short) const noexcept;
+    void _emulate_vpa(const short) const noexcept;
+    void _emulate_cnorm() const noexcept;
+    void _emulate_civis() const noexcept;
+    void _emulate_cpl(const short) const noexcept;
+    void _emulate_cnl(const short) const noexcept;
+
+    std::string do_clear() const noexcept;
+
+public:
+    TermDb() = default;
+
+    TermDb(const std::string &, std::string) = delete;
+    bool parse(const std::string, std::string);
+
+    explicit operator bool() const noexcept { return isValidState; }
+    std::string getName() const { return name; }
```

Comment:
`return "cmd.exe"` directly!

Reply:
rather I have set name to "cmd.exe" in ctor of TermDb in Windows